### PR TITLE
br: add description for --granularity

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-guide.md [hunk 1]
```diff
@@ -197,6 +199,8 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
 
 - TiDB 恢复的时候会尽可能打满 TiKV CPU、磁盘 IO、网络带宽等资源，所以推荐在空的集群上执行备份数据的恢复，避免对正在运行的业务产生影响。
 - 备份数据的恢复速度与集群配置、部署、运行的业务都有比较大的关系。在内部多场景仿真测试中，单 TiKV 存储节点上备份数据恢复速度能够达到 100 MiB/s。在不同用户场景下，快照恢复的性能和影响应以实际测试结论为准。
+- 在 TiDB v7.6 之后的版本中，通过指定 --concurrency 参数，可以控制单 TiKV 节点下载任务并发数，默认 128，适当调大该值可以让所有 TiKV 充分执行下载任务，直到打满下载带宽。
```
Comment (BornChanger) [inline br/br-snapshot-guide.md:202]: 是否要用一个新参数?

Comment (3pointer) [reply br/br-snapshot-guide.md:202]: cc @benmaoer

File: br/br-snapshot-guide.md [hunk 2]
```diff
@@ -197,6 +199,8 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
...
+
```
Comment (benmaoer) [inline br/br-snapshot-guide.md:203]:
```suggestion

    ```sql
br restore full \
    --pd "${PDIP}:2379" \
    --storage "s3://${Bucket}/${Folder}" \
    --s3.region "${region}" \
    --granularity "coarse-grained" \
    --tikv-max-restore-concurrency 128 \
    --send-credentials-to-tikv=true \
    --log-file restorefull.log
    ```
 
```

File: br/br-snapshot-guide.md (hunk 1 above)
Comment (benmaoer) [inline br/br-snapshot-guide.md:202]:
```suggestion
- 在 v7.6.0 之后的版本中，BR 提供了一个实验性特性，允许通过指定新增的命令行参数 `--granularity=coarse-grained` 启用粗细粒度结合的 Region 打散算法和新增的命令行参数 `--tikv-max-restore-concurrency` 设置单 TiKV 节点的下载任务并发度，充分利用每个 TiKV 节点的所有资源实现了并行快速恢复。在实际案例中，大规模 Region 场景下，集群快照恢复速度最高提升约 10 倍。
```

**Review by benmaoer — CHANGES_REQUESTED** [review]
add new example for br restoration acceleration

File: br/br-snapshot-guide.md [hunk 3]
```diff
@@ -71,6 +71,8 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
+在 v7.6 版本之后，BR 支持了粗粒度打散 Region 的方式来进行恢复，以避免在 Region 过多的场景中，打散 Region 成为瓶颈的问题，通过指定 --granularity="coarse-grained" 的方式即可开启。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:74]:
```suggestion
从 v7.6.0 开始，BR 支持粗粒度打散 Region 的方式来进行恢复，以避免在 Region 过多的场景中，打散 Region 成为瓶颈的问题，通过指定 `--granularity=coarse-grained` 的方式即可开启。
```

File: br/br-snapshot-guide.md [hunk 4]
```diff
@@ -71,6 +71,8 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
+从 v7.6.0 开始，BR 支持粗粒度打散 Region 的方式来进行恢复，以避免在 Region 过多的场景中，打散 Region 成为瓶颈的问题，通过指定 `--granularity=coarse-grained` 的方式即可开启。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:74]:
```suggestion
为了解决在 Region 过多的情况下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持粗粒度打散 Region 的方式进行恢复。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```

**Review by BornChanger — APPROVED** [review]

File: br/br-snapshot-guide.md [hunk 5]
```diff
@@ -71,6 +71,8 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
+为了解决在 Region 过多的情况下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持粗粒度打散 Region 的方式进行恢复。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:74]:
```suggestion
为了解决 Region 数量时，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: br/br-snapshot-guide.md [hunk 6]
```diff
@@ -197,6 +199,19 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
...
+    br restore full \
+    --pd "${PDIP}:2379" \
+    --storage "s3://${Bucket}/${Folder}" \
+    --s3.region "${region}" \
+    --granularity "coarse-grained" \
+    --tikv-max-restore-concurrency 128 \
+    --send-credentials-to-tikv=true \
+    --log-file restorefull.log
+    ```
+
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:214]:
```suggestion
```

File: br/br-snapshot-guide.md [hunk 7]
```diff
@@ -197,6 +199,18 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
...
 - TiDB 恢复的时候会尽可能打满 TiKV CPU、磁盘 IO、网络带宽等资源，所以推荐在空的集群上执行备份数据的恢复，避免对正在运行的业务产生影响。
 - 备份数据的恢复速度与集群配置、部署、运行的业务都有比较大的关系。在内部多场景仿真测试中，单 TiKV 存储节点上备份数据恢复速度能够达到 100 MiB/s。在不同用户场景下，快照恢复的性能和影响应以实际测试结论为准。
+- 从 v7.6.0 开始，BR 提供了一个实验性特性，允许通过指定命令行参数 `--granularity=coarse-grained` 启用粗细粒度结合的 Region 打散算法。此外，可以使用 `--tikv-max-restore-concurrency` 设置单 TiKV 节点的下载任务并发度，充分利用每个 TiKV 节点的所有资源实现并行快速恢复。在实际案例中，大规模 Region 场景下，集群快照恢复速度最高提升约 10 倍。使用示例如下：
+
+    ```bash
+    br restore full \
+    --pd "${PDIP}:2379" \
+    --storage "s3://${Bucket}/${Folder}" \
+    --s3.region "${region}" \
+    --granularity "coarse-grained" \
```
Comment (qiancai) [inline br/br-snapshot-guide.md:209]: L202 写的是 `--granularity=coarse-grained`，哪个是对的呢，需要改成一致的吗 (release notes 里也有这个问题）

File: br/br-snapshot-guide.md [hunk 8]
```diff
@@ -71,6 +71,8 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
+为了解决 Region 数量时，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:74]:
```suggestion
为了解决大规模 Region 场景下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持以粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```

File: br/br-snapshot-guide.md [hunk 9]
```diff
@@ -197,6 +199,18 @@ TiDB 备份功能对集群性能（事务延迟和 QPS）有一定的影响，
 
 - TiDB 恢复的时候会尽可能打满 TiKV CPU、磁盘 IO、网络带宽等资源，所以推荐在空的集群上执行备份数据的恢复，避免对正在运行的业务产生影响。
 - 备份数据的恢复速度与集群配置、部署、运行的业务都有比较大的关系。在内部多场景仿真测试中，单 TiKV 存储节点上备份数据恢复速度能够达到 100 MiB/s。在不同用户场景下，快照恢复的性能和影响应以实际测试结论为准。
+- 从 v7.6.0 开始，BR 提供了一个实验性特性，允许通过指定命令行参数 `--granularity=coarse-grained` 启用粗细粒度结合的 Region 打散算法。此外，可以使用 `--tikv-max-restore-concurrency` 设置单 TiKV 节点的下载任务并发度，充分利用每个 TiKV 节点的所有资源实现并行快速恢复。在实际案例中，大规模 Region 场景下，集群快照恢复速度最高提升约 10 倍。使用示例如下：
```
Comment (qiancai) [inline br/br-snapshot-guide.md:202]: v7.6 release notes https://github.com/pingcap/docs-cn/pull/15667/files 里写的是，`--tikv-max-restore-concurrency` 设置单 TiKV 节点的恢复任务并发度，需要两边统一下吗

File: br/br-snapshot-guide.md (hunk 7 above)
Comment (3pointer) [reply br/br-snapshot-guide.md:209]: --granularity="coarse-grained" 和 --granularity "coarse-grained" 两个都是一样的效果

File: br/br-snapshot-guide.md (hunk 9 above)
Comment (3pointer) [reply br/br-snapshot-guide.md:202]: 统一成下载任务好一点

File: br/br-snapshot-guide.md [hunk 10]
```diff
@@ -71,6 +71,8 @@ tiup br restore full --pd "${PD_IP}:2379" \
 --storage "s3://backup-101/snapshot-202209081330?access-key=${access-key}&secret-access-key=${secret-access-key}"
 ```
 
+为了解决大规模 Region 场景下，打散 Region 可能成为恢复过程中的瓶颈问题，BR 从 v7.6.0 开始支持以粗粒度打散 Region 的方式进行恢复（实验特性）。你可以通过指定 `--granularity=coarse-grained` 启用该方式。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:74]: 这里写的是 “以粗粒度打散”，L202 写的是 “粗细粒度结合的 Region 打散“，哪个更准确呢，要改成一致的吗

Comment (3pointer) [reply br/br-snapshot-guide.md:74]: fixed

**Review by qiancai — APPROVED** [review]

**Pull request merged**